tests/data/test_sampler.py

```diff
@@ -0,0 +1,99 @@
+from myrtlespeech.data.sampler import SequentialRandomSampler
+from myrtlespeech.data.sampler import SortaGrad
+
```

Comment:
These tests are copied over from `deepspeech_internal`, which is fine, but they should be updated to use  [Hypothesis](https://hypothesis.readthedocs.io/en/latest/).

Mainly, `dataset_gen`, `n_batches`, `batch_size`, `full_last_batch` etc can become parameters of each test that are generated by a search strategy. The benefit is that now failing tests search over a wider range and find the minimal failing test case. Currently the tests have arbitrary values chosen for each of the above.